index.html

```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
+            <div class="box">             
               <h4> Straightforward Affordable Pricing</h4>
               <p>Whether you are purchasing or leasing, our goal is to provide realistic solutions within your budget. Transparency is very important to us here at NEAM. Our clients will always get a fair cost-estimate before the start of any development.</p>
             </div>
           </div>
-
         </div>
-
       </div>
-    </section><!-- End Why Us Section -->
-
+    </section>
+            <!-- End Why Us Section -->
     <!-- ======= Clients Section ======= -->
    <!-- <section id="clients" class="clients">
       <div class="container" data-aos="zoom-in">
         <div class="row d-flex align-items-center">
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-1.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-2.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-3.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-4.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-5.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-6.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
         </div>
       </div>
-    
-    </section>
-  -->
-    <!-- End Clients Section -->
-
+    </section>-->
+          <!-- End Clients Section -->
    <!-- ======= Services Section ======= -->
    <section id="services" class="services">
     <div class="container">
-
       <div class="section-title">
         <span>Services</span>
         <h2>Services</h2>
       </div>
-
       <div class="row">
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch" data-aos="fade-up">
           <div class="icon-box">
-            <div class="icon"><i class="bx bxl-dribbble"></i></div>
+            <div class="icon"><i class="bi-code-square"></i></div>
             <h4><a href="">Application Development & Mobile Application Development</a></h4>
             <p>We have a strong understanding of programming languages and the ability to solve complex problems. We also are able to work well in a team and communicate effectively with others. Additionally, we have deep understanding of the latest technologies and trends in the field</p>

```

Comment:
```suggestion
            <p>We have a strong understanding of programming languages and the ability to solve complex problems. We also are able to work well in a team and communicate effectively with others. Additionally, we have deep understanding of the latest technologies and trends in the field.</p>

```

This sentence is just missing a period!